.github/workflows/dhis2-verify-app.yml

```diff
@@ -24,6 +24,9 @@ jobs:
             - uses: c-hive/gha-yarn-cache@v1
             - run: yarn install --frozen-lockfile
 
+            - name: Check types
+              run: yarn check-types
```

Comment:
Not sure if it's best to run `"tsc --noemit"` directly or refer to the script in package.json here.